storage/types.go

```diff
@@ -299,11 +304,19 @@ func (n *NodeID) Split(prefixBytes, suffixBits int) ([]byte, Suffix) {
 	a := make([]byte, len(n.Path))
 	copy(a, n.Path)
 
+	bits := n.PrefixLenBits - prefixBytes*8
+	if bits > suffixBits {
+		panic(fmt.Sprintf("storage Split: %x(n.PrefixLenBits: %v - prefixBytes: %v *8) > %v", n.Path, n.PrefixLenBits, prefixBytes, suffixBits))
+	}
+	if bits == 0 {
+		panic(fmt.Sprintf("storage Split: %x(n.PrefixLenBits: %v - prefixBytes: %v *8) == 0", n.Path, n.PrefixLenBits, prefixBytes))
+	}
+	suffixBytes := (bits + 7) / 8
```

Comment:
It lives on (at least currently): https://github.com/google/trillian/blob/d9f0b5187c3cca04fa30a9fdc63fb3afc595fda4/storage/types.go#L71